api/v1alpha1/flowcollector_types.go

```diff
@@ -215,34 +225,45 @@ type FlowCollectorKafka struct {
 	// tls client configuration.
 	// +optional
 	TLS ClientTLS `json:"tls"`
+
+	//+kubebuilder:validation:Minimum=0
+	//+kubebuilder:default:=1
+	// consumerReplicas defines the number of replicas (pods) to start for flowlogs-pipeline-transformer, which consumes Kafka messages.
+	// This setting is ignored when Kafka is disabled.
+	ConsumerReplicas int32 `json:"consumerReplicas,omitempty"`
```

Comment:
I understand the logic of moving the field here, but I think it would be better to let it in the FLP section:
- while this fields only apply when kafka is enabled, this field apply to FLP. May be we could name it better in FLP like "KafkaConsumerReplicas"
- we would be more consitant with the console plugin section which also has this kind of field

Reply:
done